### PR TITLE
[TEVA-1292] job alerts user journey 1

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-1292

## Changes in this PR:

this is part one of the Jira task which i am creating PR for so there is a environment for chris to check if he wants to. but this is mergable in it its own right i think

## Screenshots of UI changes:

Screenshot 2020-09-24 at 15 19 49
Screenshot 2020-09-24 at 15 19 35

### CODE REVIEW

**Review by cpjmcquillan — APPROVED** [review]
Do you want to update the content of the link at the same time?

**Comment by alexbowen** [comment]
> Do you want to update the content of the link at the same time?

ive changed this to WIP as this PR will have all changes for this task

File: app/views/nqt_job_alerts/new.html.haml [hunk 1]
```diff
@@ -32,6 +32,10 @@
...
 
-      = render(NotificationComponent.new(content: t('nqt_job_alerts.intro.unsubscribe'), style: 'notice', background: true, alert: true, dismiss: false))
+      = render(NotificationComponent.new(content: t('nqt_job_alerts.intro.unsubscribe'), style: 'notice', background: true, alert: 'info', dismiss: false))
 
-      = f.govuk_submit t('buttons.subscribe'), classes: 'nqt-job-alert-subscribe-gtm'
+      = f.govuk_submit t('buttons.subscribe'), classes: 'nqt-job-alert-subscribe-gtm govuk-!-padding-left-8 govuk-!-padding-right-8'
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds.govuk-body{ class: 'govuk-!-margin-top-4' }
+    = raw t('terms_and_conditions.read')
```
Comment (cpjmcquillan) [inline app/views/nqt_job_alerts/new.html.haml:41]: We should be able to lose the `raw` and suffix the translation with `_html`

File: app/views/subscriptions/new.html.haml [hunk 2]
```diff
@@ -12,33 +14,35 @@
...
+      = f.govuk_submit t('buttons.subscribe'), classes: 'govuk-!-padding-left-8 govuk-!-padding-right-8'
 
-      = f.govuk_submit t('buttons.subscribe')
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    = link_to t('buttons.cancel'), jobs_path(@subscription.search_criteria_to_h), class: 'govuk-link govuk-!-font-size-19'
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds.govuk-body{ class: 'govuk-!-margin-top-4' }
+    = raw t('terms_and_conditions.read')
```
Comment (cpjmcquillan) [inline app/views/subscriptions/new.html.haml:48]: Same as above comment

File: app/views/subscriptions/new.html.haml [hunk 3]
```diff
@@ -12,33 +14,35 @@
...
         label: { text: t('subscriptions.labels.email_html'), size: 's' },
         hint_text: t('subscriptions.hints.email'),
         required: true
 
-      = f.govuk_text_field :reference,
-        label: { text: t('subscriptions.labels.reference_html'), size: 's' },
-        hint_text: t('subscriptions.hints.reference'),
-        required: true
+      = f.govuk_collection_radio_buttons :frequency,
+        [['daily', t('subscriptions.frequency.daily')], ['weekly', t('subscriptions.frequency.weekly')]],
```
Comment (cpjmcquillan) [inline app/views/subscriptions/new.html.haml:31]: These options should be defined somewhere else probably, maybe on the subscription model as `FREQUENCY_OPTIONS`

File: app/views/subscriptions/new.html.haml [hunk 4]
```diff
@@ -12,33 +14,35 @@
 
       = f.hidden_field :search_criteria
 
-      .govuk-inset-text
+      .grouped-text-block
```
Comment (cpjmcquillan) [inline app/views/subscriptions/new.html.haml:17]: The padding at the top/bottom seems a bit uneven

File: spec/system/job_seekers_can_subscribe_to_a_job_alert_spec.rb [hunk 5]
```diff
@@ -60,22 +60,21 @@
         expect(page).to have_content(I18n.t('subscriptions.confirmation.header'))
       end
 
-      scenario 'when no reference is set' do
-        visit new_subscription_path(search_criteria: { keyword: 'test' })
-        fill_in 'subscription[email]', with: 'jane.doe@example.com'
-        click_on 'Subscribe'
+      context 'and is redirected to the confirmation page' do
```
Comment (cpjmcquillan) [inline spec/system/job_seekers_can_subscribe_to_a_job_alert_spec.rb:63]: the context should be `when alert frequency is daily` and `when alert frequency is weekly`, and the scenarios `redirects to the confirmation page`

Comment (cpjmcquillan) [reply spec/system/job_seekers_can_subscribe_to_a_job_alert_spec.rb:63]: https://www.betterspecs.org/

**Review by cpjmcquillan — APPROVED** [review]

File: app/views/subscriptions/confirm.html.haml [hunk 6]
```diff
@@ -3,23 +3,27 @@
...
-      %h1.govuk-panel__title= t('subscriptions.confirmation.header')
+      %span.hod-checkmark
+        %svg{ xmlns: "http://www.w3.org/2000/svg", role: "presentation", focusable: "false", viewBox: "0 0 25 25", height: "40", width: "40" }
+          %path{ fill: "currentColor", d: "M25,6.2L8.7,23.2L0,14.1l4-4.2l4.7,4.9L21,2L25,6.2z" }
+      %h1.govuk-panel__title{ class: 'govuk-!-font-size-36' }= t('subscriptions.confirmation.header')
       .govuk-panel__body
-        %strong= t('subscriptions.confirmation.reference', reference: @subscription.reference)
-
-    %p.govuk-body=t('subscriptions.confirmation.receipt_confirmation', email: @subscription.email)
+        %h2{ class: 'govuk-!-font-size-24' }=t('subscriptions.confirmation.receipt_confirmation', email: @subscription.email)
```
Comment (david-mears-dfe) [inline app/views/subscriptions/confirm.html.haml:11]: I believe this font-size is 36 (also for the nqt confirmation page)

**Review by david-mears-dfe — CHANGES_REQUESTED** [review]

**Pull request merged**